index.html

```diff
@@ -0,0 +1,1164 @@
+              <a class="navigation-link" href="#">Vantage</a>
+              <span class="border-right"></span>
+              <a class="navigation-link" href="#">Wather</a>
+          </div>
+
+          <form class="col f-inline">
+            <div class="container-inline">
+              <input class="fm-control" type="search" placeholder="Search" aria-label="Search">
+              <i class="fas fa-search search-icon"></i>
+            </div>            
+          </form>
+
+          <div class="box-1 position-absolute"></div>
+        </div>           
+    </div>
+
+
+  <main class="container-fluid main">
+  
+
+    <div class="row text-left top-section">
+
+      <div class="col-md-5 col-lg left-content">
+        <p class="h-title">FEATURED STORIES</p>
+        <div class="featured-stories">
+          
+          <a href="#"><img class="img-top" src="images/tony-blair-joe-biden-2020-donald-trump.webp" alt="img"></a>
+          <a href="#" class="body-header-a"><p class="body-header">Class Action Lawsuits Related to Coronavirus Spike Across the Country</p></a>
+          <p>As employers adjust to new work-from-home requirements, new 
+            liabilities can arise that human resources 
+            managers have not traditionally had to consider.
+          </p>
+        </div>
+
+        <div class="featured-stories">
+          
+          <a href="#"><img class="img-top" src="images/tony-blair-joe-biden-2020-donald-trump.webp" alt="img"></a>
+          <a href="#" class="body-header-a"><p class="body-header">Class Action Lawsuits Related to Coronavirus Spike Across the Country</p></a>
+          <p>As employers adjust to new work-from-home requirements, new 
+            liabilities can arise that human resources 
+            managers have not traditionally had to consider.
+          </p>
+        </div>
+
+        <div class="featured-stories">
+          
+          <a href="#"><img class="img-top" src="images/tony-blair-joe-biden-2020-donald-trump.webp" alt="img"></a>
+          <a href="#" class="body-header-a"><p class="body-header">Class Action Lawsuits Related to Coronavirus Spike Across the Country</p></a>
+          <p>As employers adjust to new work-from-home requirements, new 
+            liabilities can arise that human resources 
+            managers have not traditionally had to consider.
+          </p>
+        </div>
+
+        <div class="featured-stories">
+          
+          <a href="#"><img class="img-top" src="images/tony-blair-joe-biden-2020-donald-trump.webp" alt="img"></a>
+          <a href="#" class="body-header-a"><p class="body-header">Class Action Lawsuits Related to Coronavirus Spike Across the Country</p></a>
+          <p>As employers adjust to new work-from-home requirements, new 
+            liabilities can arise that human resources 
+            managers have not traditionally had to consider.
+          </p>
+        </div>
+
+        <div class="card" style="width: auto;">
```

Comment:
You have set this here: `style="width: auto"`. Kindly move it to your CSS file, and do so to all other similar inline styles.